sdk/storage/azure-storage-queue/azure/storage/queue/aio/_queue_client_async.py

```diff
@@ -94,14 +94,13 @@ def __init__(
         super(QueueClient, self).__init__(
             account_url, queue_name=queue_name, credential=credential, loop=loop, **kwargs
         )
-        self._client = AzureQueueStorage(self.url, base_url=self.url,
-                                         pipeline=self._pipeline, loop=loop)
-        self._client._config.version = get_api_version(kwargs)  # pylint: disable=protected-access
+        self._client_async = AzureQueueStorage(self.url, base_url=self.url, pipeline=self._pipeline, loop=loop)  # type: ignore
```

Comment:
Do you need this type ignore here?

Reply:
Good catch, I think that was leftover from before we came up with renaming the client 😄